src/components/PeopleManagement/tests/CreateGroupModal.test.jsx

```diff
@@ -129,6 +129,7 @@ const CreateGroupModalWrapper = ({
 
 describe('<CreateGroupModal />', () => {
```

Comment:
I would love to have a test against this bug that is red when a new CSV upload does not re-add unique emails, and green after this change. That would prevent regressions where someone breaks this again in the future and we do not notice.